railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -224,6 +224,18 @@ def initialize(*args)
           raise Error, "Invalid value for --database option. Supported for preconfiguration are: #{DATABASES.join(", ")}."
         end
 
+        # --webpack uses webpacker, which uses action view helpers
+        # --api skips action view
+        if options[:webpack] && options[:api]
+          raise Error, "Incompatible options --api and --webpack supplied. Please use either --api or --webpack"
+        end
+
+        # --webpack requires webpacker gem to be pre-bundled
+        # --skip-bundle skips bundle install
+        if options[:webpack] && options[:skip_bundle]
+          raise Error, "Incompatible options --webpack and --skip-bundle supplied. Please use either --skip-bundle or --webpack"
```

Comment:
Should we fail in this case or just skip the `wepacker:install` call that we do later? I always generate my applications with `--skip-bundler` (it is in my ~/.railsrc) and I'm fine with adding webpacker in my gemfile when using `--webpack` and running `webpacker:install` manually. But the fact that I can't do that when we merge this code annoys me.